app/views/content/github/_review_checklist.erb

```diff
@@ -25,11 +24,13 @@
 - [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
 - [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
 - [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
-- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
+- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
 - [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
 
 ### Software paper
 
-- [ ] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
```

Comment:
We can't guarantee that we will have access to affiliation information using ORCID as this very much depends on whether the user has added this information to their profile _and_ whether the allow this information to be read publicly via the API (the default I think is to *not* allow this). 

So, I'm happy to remove this if we think that the `Contribution and authorship` question covers the authorship question.

Reply:
I think so. I also feel this is perhaps a point the editors should check rather than the reviewers.